multi_label_trainer_tfidf.ipynb

```diff
@@ -4,10 +4,9 @@
   "metadata": {
```

Comment:
<pre class="ql-syntax" spellcheck="false">history = shallow_mlp_model.fit(train_dataset, validation_data=validation_dataset, epochs=epochs)

def plot_result(item):
    plt.plot(history.history[item], label=item)
    plt.plot(history.history["val_" + item], label="val_" + item)
    plt.xlabel("Epochs")
    plt.ylabel(item)
    plt.title("Train and Validation {} Over Epochs".format(item), fontsize=14)
    plt.legend()
    plt.grid()
    plt.show()


plot_result("loss")
plot_result("categorical_accuracy")
</pre><p> </p><p>We should add plots for the training history.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>